src/persist-types/src/ord.rs

```diff
@@ -44,6 +44,30 @@ pub enum ColOrd<'a> {
 }
 
 impl<'a> ColOrd<'a> {
+    /// Returns the index of the (min, max) values in the column.
+    pub fn minmax(&self, len: usize) -> (usize, usize) {
```

Comment:
nit: `fn min_max`

Reply:
also, very interesting idea for how to structure this! I'd been imagining it as something that happened while we build the part, but I can't see any reason that we'd prefer that over this (plus it's probably hard given the interface) :clap: